web_src/css/repo.css

```diff
@@ -1142,12 +1142,18 @@
 .repository.view.issue .comment-list .event .detail {
   margin-top: 4px;
   margin-left: 14px;
+  display: flex;
+  align-items: center;
 }
 
 .repository.view.issue .comment-list .event .detail .svg {
   margin-right: 2px;
 }
 
+.repository.view.issue .comment-list .event .detail span{
+  display: flex;
+}
```

Comment:
Sorry but it doesn't look good to me.

Putting `flex` on a general `span` tag means its contents lose spaces between elements.

I think it's better use some more explicit styles for the fix.

Reply:
Did some improves and added some new screenshots.